panther_analysis_tool/analysis_utils.py

```diff
@@ -47,6 +48,56 @@ def __init__(self, file_name: str, dir_name: str, analysis_spec: Dict[str, Any])
         self.analysis_spec = analysis_spec
 
 
+@dataclasses.dataclass
+class ClassifiedAnalysisContainer:
```

Comment:
This is the best name I could think of. I am open to suggestions if you can think of a better one!